pkg/apis/openebs.io/ndm/v1alpha1/blockdeviceclaim_types.go

```diff
@@ -56,17 +56,12 @@ const (
 )
 
 // DeviceClaimRequirements defines the request by the claim, eg, Capacity, IOPS
-type DeviceClaimRequirements struct {
+type DeviceClaimResources struct {
 	// Requests describes the minimum resources required. eg: if storage resource of 10G is
 	// requested minimum capacity of 10G should be available
-	Requests v1.ResourceList `json:"requests"`
+	Requests corev1.ResourceList `json:"requests"`
 }
 
-const (
-	// ResourceCapacity defines the capacity required as v1.Quantity
-	ResourceCapacity v1.ResourceName = "capacity"
```

Comment:
@prateekpandey14  we should define our own `ResourceStorage`. This should be same as per NDM.

Reply:
Yes done